src/Bravo3/Orm/Services/AbstractManagerUtility.php

```diff
@@ -87,4 +93,75 @@ protected function getEntityId($entity)
         $reader   = new Reader($metadata, $entity);
         return $reader->getId();
     }
+
+    /**
+     * Get the table name for a given entity class
+     *
+     * @param string $class
+     * @return string
+     */
+    protected function getTableForClass($class)
```

Comment:
Hmm, not what it's intended for (will discuss that with you in Slack).

That function is a helper for the managers sub-services - it just pulls entity metadata and gets the table name - you want the opposite. This problem though (which is the refs problem) is what this PR is heading down the direction of fixing. Although I'm still not sure how to support annotation mappings with the ref problem fixed (Yaml mappings can do it safely).